docker-compose.yml

```
@@ -33,10 +33,13 @@ services:
   redis:
     image: 'redis:alpine'
     # ports:
-    #   - target: 6379
-    #     published: 6379
-    #     protocol: tcp
+    #   - host_ip: 127.0.0.1
     #     mode: host
+    #     protocol: tcp
+    #     # Note that we expose redis on a non-standard port to avoid
+    #     # conflict with the Admiral.
```

Include a link to the Admiral installation role for further context.
```suggestion
    #     # conflict with the Admiral.
    #     # https://github.com/cisagov/ansible-role-admiral
```